ConsentViewController/Classes/SourcePointClient/SourcePointClient.swift

```diff
@@ -225,12 +231,10 @@ class SourcePointClient: SourcePointProtocol {
             "propertyId": propertyId,
             "messageId": messageId
         ])! // swiftlint:disable:this force_unwrapping
```

Comment:
do we still need this swiftlint:disable ?

Reply:
I just don't see any force unwrapping in the next line

Reply:
We're unwrapping the result of `appendQueryParams` (just behind the comment). Btw, I also had a hard time to spot it.